tx_test.go

```diff
@@ -9,11 +9,12 @@ import (
 	"testing"
 
 	bolt "go.etcd.io/bbolt"
+	"go.etcd.io/bbolt/internal/btesting"
 )
 
 // TestTx_Check_ReadOnly tests consistency checking on a ReadOnly database.
 func TestTx_Check_ReadOnly(t *testing.T) {
-	db := MustOpenDB()
+	db := btesting.MustCreateDB(t)
 	defer db.Close()
```

Comment:
Close has been handled by t.Cleanup. I think we can consider to delete this line.

Reply:
Done